Banking/src/main/java/com/example/Banking/service/AccountDaoImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.example.Banking.service;
+import com.example.Banking.exception.IdNotFoundException;
+import com.example.Banking.model.Account;
+import com.example.Banking.repository.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+
+@Component
+public class AccountDaoImpl implements AccountDao {
+
+    @Autowired
+    private AccountRepository accountRepository;
+
+    String msg="Account is not Found for this ";
```

Comment:
A constant should be declared as static and final with upper cases. There should be space before and after '=' sign for better readability. The variable should be named for the purpose its created. msg doesn't match with the purpose its serving

Reply:
Sure. I will change it